server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/tservers.js

```diff
@@ -183,29 +112,155 @@ function refreshDeadTServersTable() {
- */
-function refreshRecoveryList() {
-  $('#recovery-caption').hide(); // Hide the caption about highlighted rows on each refresh
-  getRecoveryList().then(function () {
-    recoveryList = []
-    var data = sessionStorage.recoveryList === undefined ?
-      [] : JSON.parse(sessionStorage.recoveryList);
-    data.recoveryList.forEach(entry => {
-      recoveryList.push(entry.server);
+    refreshRecoveryList();
+
+    // Create a table for tserver list
+    tserversTable = $('#tservers').DataTable({
+        "ajax": {
+            "url": '/rest/tservers',
+            "dataSrc": "servers"
+        },
+        "stateSave": true,
+        "columnDefs": [
+            {
+                "targets": "big-num",
+                "render": function (data, type) {
+                    if (type === 'display') {
+                        data = bigNumberForQuantity(data);
+                    }
+                    return data;
+                }
+            },
+            {
+                "targets": "duration",
+                "render": function (data, type) {
+                    if (type === 'display') {
+                        data = timeDuration(data);
+                    }
+                    return data;
+                }
+            },
+            {
+                "targets": "percent",
+                "render": function (data, type) {
+                    if (type === 'display') {
+                        data = Math.round(data * 100) + '%';
+                    }
+                    return data;
+                }
+            }
+        ],
+        "columns": [
+            {
+                "data": "hostname",
+                "type": "html",
+                "render": function (data, type, row) {
+                    if (type === 'display') {
+                        data = '<a href="/tservers?s=' + row.id + '">' + row.hostname + '</a>';
+                    }
+                    return data;
+                }
+            },
+            { "data": "tablets" },
+            { "data": "lastContact" },
+            { "data": "responseTime" },
+            { "data": "entries" },
+            { "data": "ingest" },
+            { "data": "query" },
+            { "data": "holdtime" },
+            { "data": "scansCombo" },
+            { "data": "minorCombo" },
+            { "data": "majorCombo" },
+            { "data": "indexCacheHitRate" },
+            { "data": "dataCacheHitRate" },
+            { "data": "osload" }
+        ],
+        "rowCallback": function (row, data, index) {
+            // reset background of each row
+            $(row).css('background-color', '');
+
+            // return if the current row's tserver is not recovering
+            if (!recoveryList.includes(data.hostname)) {
+                return;
+            }
+
+            // only show the caption if we know there are rows in the tservers table
+            $('#recovery-caption').show();
```

Comment:
You could put this in an else above.

Reply:
Oh wait nvm JSlint probably told you not to.

Reply:
Looks like jslint doesn't complain about the changes made in [086031e](https://github.com/apache/accumulo/pull/2685/commits/086031ecff58f4c089efb9eaa257218723a1553a)